code/modules/mob/living/carbon/human/species_types/flypeople.dm

```diff
@@ -7,7 +7,6 @@
 	meat = /obj/item/food/meat/slab/human/mutant/fly
 	mutanteyes = /obj/item/organ/eyes/fly
 	liked_food = GROSS
-	disliked_food = CLOTH
```

Comment:
this will make flypeople inherit the disliked_food = GROSS of /datum/species.

the correct fix here is to just replace "CLOTH" in this line with "NONE".

Reply:
should be fixed now